src/Routers/UsersRouter.js

```diff
@@ -267,6 +267,77 @@ export class UsersRouter extends ClassesRouter {
     });
   }
 
+  handleVerifyPassword(req) {
```

Comment:
@johnnydimas this whole section could be factored differently, as this is shared 100% with the login code right? 

We could probably have a function somewhere that handles password validation, in both login / verifyPassword and only create the session on login? What do you think?

Reply:
That sounds great to me!  Good call on removing the POST handler as well, it's not necessary for this. 

I pushed up a fix removing the POST now, but I will update the pull request again with a helper to facilitate both login and verify password. 

Thanks for looking it over!